Ray.jl/src/runtime.jl

```diff
@@ -30,6 +41,16 @@ function init()
         return nothing
     end
 
+    if isnothing(runtime_env)
+        # Set default for `JOB_RUNTIME_ENV` when `Ray.init` is called before `@ray_import`.
+        # This ensures a call to `@ray_import` after `Ray.init` will fail.
```

Comment:
this isn't really accurate given that the `@ray_import` code that sets this ref is run at parse time (it's in the macro body, not the returned expression) not run time, right?

Reply:
You are correct the comment isn't accurate. When running in the REPL however the macro can be expanded after `Ray.init`

Reply:
Comment is now correct given that we set `JOB_RUNTIME_ENV` at runtime